spec/support/shared/protocol.rb

```diff
@@ -5,7 +5,8 @@
     describe 'length' do
       let(:field) { bytes.to_s[0..3] }
       it 'serializes the length' do
-        expect(field).to be_int32(bytes.length)
+        bytes.rewind!
```

Comment:
If this is only needed for JRuby can it be conditioned on `BSON::Environment.jruby?` ?

Reply:
Sure thing!